Dynavity/Dynavity/model/CanvasElementProtocol.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import CoreGraphics
+
+protocol CanvasElementProtocol {
```

Comment:
I think this can also be `protocol CanvasElementProtocol: Identifiable`? So we don't have to manually conform to Identifiable in the individual canvas elements

Reply:
Yep this makes sense. Will change it, thanks!

Reply:
Nvm, I realise this doesn't work